docs/pages/contributing.md

```diff
@@ -1,58 +1,96 @@
 ---
-title: Contributing
-description: This is a documentation page.
+title: Contributing to a Project
```

Comment:
Let's keep this as `Contributing` as it is pretty standard across all open source projects.